Cargo.toml

```diff
@@ -45,6 +45,7 @@ criterion = "0.3.5"
 rand_xoshiro = "0.6.0"
 ascii_table = "4.0.2"
 num-format = "0.4.0"
+libc = "0.2.127"
```

Comment:
You should probably feature gate this under `[target.'cfg(unix)'.dev-dependencies]` as well as usages of libc

Reply:
Thanks - yeah, I tried to see exactly [what was supported on libc for windows](https://docs.rs/libc/latest/i686-pc-windows-msvc/libc/index.html), but it's only the `timeval` type out of the things I'm using (no rusage of course).